frontend/src/components/Client/ReviewsCarousel/ReviewsCarousel.js

```diff
@@ -18,6 +27,8 @@ const ReviewsCarousel = ({ content }) => {
           items={content}
           responsive={responsive}
           controlsStrategy="default"
+          renderPrevButton={renderPrevButton}
+          renderNextButton={renderNextButton}
         />
```

Comment:
What was wrong with the previous implementation for this? If its honestly too much of a hassle to have the same package play nice, I'm not against just installing a separate specialized package that handles larger viewport carousels better and just use `react-alice-carousel` for this card specifically

Reply:
The rendering functions allow customization of the prev/next buttons. The default ones have been replaced by font-awesome icons in both carousels.